components/style/properties/data.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, name, values, gecko_constant_prefix=None,
         self.extra_gecko_aliases = parse_aliases(extra_gecko_aliases or "")
         self.extra_servo_aliases = parse_aliases(extra_servo_aliases or "")
         self.consts_map = {} if custom_consts is None else custom_consts
-        self.gecko_strip_moz_prefix = gecko_strip_moz_prefix
+        self.gecko_strip_moz_prefix = True \
```

Comment:
I tried adding `gecko_strip_moz_prefix` to `keyword_kwargs` without changing this, and it failed to build because the moz prefix is incorrectly kept in many other properties, so this is necessary.

Alternatively, we can change how `keyword_kwargs` is generated, use `pop('gecko_strip_moz_prefix', True)` for this instead of just adding it to the list. But having different places maintaining a single default value seems fragile.